addons/mail/static/src/components/message_reaction_group/tests/message_reaction_group_tests.js

```diff
@@ -0,0 +1,165 @@
+/** @odoo-module **/
+
+import { insert, insertAndReplace, link, replace } from '@mail/model/model_field_command';
+import { makeDeferred } from '@mail/utils/deferred/deferred';
+import {
+    afterEach,
+    afterNextRender,
+    beforeEach,
+    createRootMessagingComponent,
+    nextAnimationFrame,
+    start,
+} from '@mail/utils/test_utils';
+
+QUnit.module('mail', {}, function () {
+QUnit.module('components', {}, function () {
+QUnit.module('message_reaction_group', {}, function () {
+QUnit.module('message_reaction_group_tests.js', {
+    beforeEach() {
+        beforeEach(this);
+
+        this.start = async params => {
+            const res = await start({ ...params, data: this.data });
+            const { afterEvent, components, env, widget } = res;
+            this.afterEvent = afterEvent;
+            this.components = components;
+            this.env = env;
+            this.widget = widget;
+            return res;
+        };
+    },
+    afterEach() {
+        afterEach(this);
+    },
+});
+
+QUnit.test('message should have a reaction displayed', async function (assert) {
+    assert.expect(3);
+
+    this.data['mail.channel'].records.push({ id: 10 });
+    this.data['mail.message'].records.push({
+        body: 'not empty',
+        id: 10,
+        model: 'mail.channel',
+        res_id: 11,
+    });
+    this.data['mail.message.reaction'].records.push({
+        content: '😊',
+        message_id: 10,
+        partner_id: 7,
+    });
+    const { createThreadViewComponent } = await this.start();
+    const threadViewer = this.messaging.models['ThreadViewer'].create({
+        hasThreadView: true,
+        qunitTest: insertAndReplace(),
+        thread: insert({
+            id: 11,
+            model: 'mail.channel',
+        }),
+    });
+    await createThreadViewComponent(threadViewer.threadView);
```

Comment:
use real flux instead of take thread viewer (in general: open discuss on channel, create chatter container, or open chat window from messaging menu, depending on what is easiest/most appropriate)